db/pom.xml

```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
```

Comment:
The liquibase deploy is currently using mvn to update. Could change it to sbt potentially or keep it like this to keep the dependencies separate? idk

Reply:
This works for me

Reply:
Ultimately I think it would be good to consolidate both the dependencies and required build tools but this is okay for now.